SPEC.md

```diff
@@ -533,3 +533,4 @@ The `dns` field contains a dictionary consisting of common DNS information.
 ## Well-known Error Codes
 - `1` - Incompatible CNI version
 - `2` - Unsupported field in network configuration. The error message must contain the key and value of the unsupported field.
+- `100` - Error during invocation of cni plugin. Plugin didn't specify the error code.
```

Comment:
I would say something like:

`Generic error during invocation of a plugin. Used when the plugin does not specify the error code.  Can also be used by frameworks and libraries that implement the CNI specification to indicate errors that occur before the plugin is actually invoked.`